### PR TITLE
fix: Changes the Loss of signal from 6min to 10min.

### PR DESCRIPTION
**Pull request by farhaanbukhsh**
This change is done to avoid many false alert that is generated and recovers in below 10mins.

**JIRA tickets**: [SE-6314](https://tasks.opencraft.com/browse/SE-6314)

~~**Discussions**: Link to any public dicussions about this PR or the design/architecture. Otherwise omit this.~~

**Dependencies**: None

~~**Screenshots**: Always include screenshots if there is any change to the UI.~~

~~**Sandbox URL**: TBD - sandbox is being provisioned.~~

~~**Merge deadline**: "None" if there's no rush, "ASAP" if it's critical, or provide a specific date if there is one.~~

**Testing instructions**:

The policy is already applied in the new relic dashboard.

### CODE REVIEW

**Comment by itsjeyd** [comment]
@farhaanbukhsh Drive-by question: Does this PR replace #1? 

CC @Cup0fCoffee @gabor-boros

**Comment by farhaanbukhsh** [comment]
@itsjeyd  yes :)

**Comment by itsjeyd** [comment]
@farhaanbukhsh OK thanks, I closed that one.

CC @Cup0fCoffee @gabor-boros

**Comment by farhaanbukhsh** [comment]
@itsjeyd  thanks :) 

@ArturGaspar  can you please review this :) cc: @0x29a

File: tutornewrelic/newrelic/client.py [hunk 1]
```diff
@@ -297,7 +297,7 @@ def create_alert_condition(
                     },
                 ],
                 "expiration": {
-                    "expirationDuration": 120,
```
Comment (ArturGaspar) [inline tutornewrelic/newrelic/client.py:300]: @farhaanbukhsh I don't understand what this is doing. This looks like it is changing something from 2 minutes to 10 minutes, but the PR title says 6 minutes to 10 minutes.

Comment (farhaanbukhsh) [reply tutornewrelic/newrelic/client.py:300]: @ArturGaspar Initially it was 2 min then https://github.com/open-craft/tutor-contrib-newrelic/pull/1 changes it to 6min and then we finally changed it to 10 mins. Hence I wrote 6 to 10, sorry for the confusion caused.

**Comment by ArturGaspar** [comment]
:+1:
- [x] I tested this: Checked the test alerts
- [x] I read through the code
- [ ] I checked for accessibility issues - N/A
- [ ] Includes documentation
- [ ] I made sure any change in configuration variables is reflected in the corresponding client's `configuration-secure` repository. - N/A

**Review by ArturGaspar — APPROVED** [review]

**Pull request merged**